settings.json

```diff
@@ -0,0 +1,600 @@
+{
+    "type": "object",
+    "properties": {
+        "ALLOWED_HOSTS": {
+            "type": "array",
+            "default": []
+        },
+        "DATABASES": {
```

Comment:
Bearing in mind #5099, at some point we'll need to have an appropriately populated `description` for each of these properties in the schema.